javascripts/discourse/widgets/layouts-category-list.js

```diff
@@ -186,6 +190,54 @@ export default layouts.createLayoutsWidget('category-list', {
   }
 });
 
+createWidget('layouts-minimize-categories', {
+  tagName: 'li.layouts-minimize-button.layouts-category-link',
+  buildKey: (attrs) => 'layouts-minimize-categories',
+  
+  defaultState(attrs) {
+    return {
+      sidebarMinimized: false
+    }
+  },
+
+  html(attrs, state) {
```

Comment:
Considering we're always going to have the same result here, we should perhaps simplify this a bit, i.e.

```
html() {
   return [
      h('div.category-logo.minimize-icon', iconNode('chevron-circle-left')),
      h('div.category-name.minimize-text', I18n.t(themePrefix("minimize_button_label")))
   ]
}
```

We can always add in more logic later, however if you look at the code now you're first given the impression there must be some other logic there, but it turns out it's always the same two elements.